hazelcast/src/test/java/com/hazelcast/jet/impl/submitjob/memberside/JobUploadStatusTest.java

```diff
@@ -194,4 +195,18 @@ public void testChangeLastUpdatedTime() {
         jobUploadStatus.changeLastUpdatedTime();
         assertEquals(fixedInstant, jobUploadStatus.lastUpdatedTime);
     }
+
+    @Test
+    public void testGetJarPath() throws IOException {
+        Path jarPath = null;
+        try {
+            when(jobMetaDataParameterObject.getUploadDirectoryPath()).thenReturn(".");
```

Comment:
Should not we rather create it in `target` dir?

Reply:
rev : 1fa7fb6b030ec9e42b367543173080a0be06e0f0